hel_django_auditlog_extra/README.md

```diff
@@ -0,0 +1,330 @@
+>
+> Docs: https://www.django-rest-framework.org/.
+
+Django REST framework is a powerful and flexible toolkit that makes it easier to build Web APIs using the Django framework.
+
+## FAQ
+
+There have been some incompatibility issues with `django-auditlog` and `django-graphene`. Some solutions to that and answers to some other common questions, issues and more details, see the [FAQ.md](./docs/FAQ.md).
+
+## Installation
+
+**Dependencies (TODO):**
+For actor handling, django-auditlog and test usage:
+
+- **`django.contrib.auth`**: Django built-in
+- **`django.contrib.contenttypes`**: Django built-in
+- **`auditlog`**: django-auditlog
+
+**`settings.py`**:
+
+```python
+INSTALLED_APPS = [
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "auditlog",
+    # ...
+    "hel_django_auditlog_extra.apps.HelDjangoAuditLogExtraConfig",
+```
+
+## Features
+
+### Context manager
+
+Code reference: [context.py](./context.py).
+
+#### `set_request_path`
+
+    Store the request path in the LogEntry's `additional_data` field.
+
+    This context manager uses a ContextVar to store the request path and
+    connects a signal receiver to automatically add it to LogEntry instances.
+    It uses a unique signal_duid to prevent duplicate signals when nested.
+
+NOTE: This is used by the [`AuditlogMiddleware`](#middleware).
+
+### Middleware
+
+Code reference: [middleware.py](./middleware.py).
+
+#### `AuditlogMiddleware`
+
+    Extends the `auditlog.middleware.AuditlogMiddleware` to fix an issue
+    with setting the actor in the audit log context.
+
+    This middleware extends the `auditlog.middleware.AuditlogMiddleware` to
+    address a potential issue where the actor (user and their IP address)
+    might not be available when `auditlog` attempts to access it.
+
+    It achieves this by explicitly setting the actor in the
+    audit log context before the request is processed. This ensures that
+    audit logs accurately reflect the user responsible for each action.
+
+    This fix is based on the suggestion provided in:
+    https://github.com/jazzband/django-auditlog/issues/115#issuecomment-1682234986
+
+    Additionally, this middleware sets the request path in the audit log
+    context, providing more context for each logged action.
+
+To use, add `"hel_django_auditlog_extra.middleware.AuditlogMiddleware"` to the list of the middlewares in `settings.py`, (instead of the one that `django-auditlog` offers):
+
+```python
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    # ...other middlewares that manipulates request context...
+    "hel_django_auditlog_extra.middleware.AuditlogMiddleware",
+]
+```
+
+### Graphene Decorators
+
+Code reference: [graphene_decorators.py](./graphene_decorators.py).
+
+#### `auditlog_access`
+
+    Decorator to init audit logging to a Graphene DjangoObjectType's get_node method.
```

Comment:
```suggestion
    Decorator to add audit logging to a Graphene DjangoObjectType's get_node method.
```
At least for me "add audit logging" is more clear here than "init audit logging". How do you see this?